### PR TITLE
Concentric

### PR DESCRIPTION
**Pull request by Tiross**
**Which issue, if any, does this resolve?**
#495

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (jwilsson) [inline .editorconfig]: Not really sure why this file was added.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
 # Changelog
+
+## Unreleased
```
Comment (jwilsson) [inline CHANGELOG.md]: Thanks, but I'll update the changelog on release.

File: lib/linters/property_ordering.js [hunk 3]
```diff
@@ -8,12 +8,20 @@ module.exports = {
...
-        if (config.style !== 'alpha') {
+        let ordering;
+
+        try {
+            ordering = require('../config/ordering/' + config.style);
+        } catch (excep) {
             throw new Error(`Invalid setting value for propertyOrdering: ${ config.style }`);
         }
 
+        if (config.style !== 'alpha') {
```
Comment (jwilsson) [inline lib/linters/property_ordering.js]: I think we can remove this check and use `util.format` when reporting the error instead. Take a look at `stringQuotes` for example.

File: .editorconfig (hunk 1 above)
Comment (Tiross) [reply .editorconfig]: My bad, it's not related to the topic, but I think it's not a bad idea, it change IDE behaviour to match what is in the file. 

Example, I prefer indent with 2 spaces, with that my Atom automatically change for 4 spaces for this project.

File: CHANGELOG.md (hunk 2 above)
Comment (Tiross) [reply CHANGELOG.md]: Removed

File: lib/linters/property_ordering.js (hunk 3 above)
Comment (Tiross) [reply lib/linters/property_ordering.js]: Yep, seems better

File: .editorconfig (hunk 1 above)
Comment (jwilsson) [reply .editorconfig]: That's true. Let's leave it in then!

**Comment by jwilsson** [comment]
Thanks for contributing! 🎉 

I'll get a new version out straight away.

**Pull request merged**